modules/system/defaults/finder.nix

```diff
@@ -96,5 +96,13 @@ with lib;
       '';
     };
 
+    system.defaults.finder.NewWindowTarget = mkOption {
+      type = types.nullOr types.str;
+      default = null;
+      description = ''
+        Change the default folder shown in Finder windows. Use "PfHm" to default to the home folder.
+        The default is unset ("Recents").
+      '';
+    };
   };
```

Comment:
Could you also add `NewWindowTargetPath`? I'm assuming that's the `defaults` option when setting `NewWindowTarget` to `Other`

Reply:
Done